static/app/components/events/eventEntries.tsx

```diff
@@ -446,9 +482,16 @@ function Entries({
     return null;
   }
 
+  if (
```

Comment:
there's still `entries` defined on line 479. you should probably delete or update them to avoid confusion